components/list/list-item-mixin.js

```diff
@@ -125,6 +125,13 @@ export const ListItemMixin = superclass => class extends composeMixins(
 				position: absolute;
 				width: 100%;
 			}
+			:host([_show-add-button][_has-nested-list]) [slot="before-content"] {
+				border-bottom: 1px solid var(--d2l-color-mica);
+				margin-bottom: -1px;
+			}
+			:host([_show-add-button][_has-nested-list]:not([selection-disabled]):not([skeleton])[selected]) [slot="before-content"] {
+				border-bottom-color: #b6cbe8;
```

Comment:
I guess we couldn't use one of our palette colors here eh?

Reply:
Oh I see this is already used. `/* celestine alpha 0.3 */`

Reply:
Yup! I'll add a comment to indicate that as in the other places